### ISSUE TITLE
What does STD and MEAN do?

### ISSUE THREAD

**Issue by brentnallt** [lars76/object-localization#3]
Hi, I'm using your code as a jumping off point for learning about localiser development, and was just a bit confused as to what STD and MEAN are explicitly used for? I think it looks like some kind of normalisation operation, but not 100% sure. Any info you can give would be greatly appreciated.
Cheers.

**Comment by lars76** [comment]
Hi, yeah you're right. In machine learning, this is called feature scaling (https://en.wikipedia.org/wiki/Feature_scaling). It is mostly used to get a better convergence.

Btw instead of manually calculating std and mean, you can also use `from keras.applications.... import preprocess_input`. I had hoped for a faster convergence speed by calculating the exact std/mean, but it seems `preprocess_input` works just as well. Hope this helps. -- Lars
